view/src/components/surplus.js

```diff
@@ -467,27 +543,44 @@ class Surplus extends Component {
                       </Box>
                     </CardContent>
                     <CardActions>
-                      <Button
-                        size="small"
-                        color="primary"
-                        onClick={() => this.handleViewOpen({ surplus })}
-                      >
-                        View
-                      </Button>
-                      <Button
-                        size="small"
-                        color="primary"
-                        onClick={() => this.handleEditClick({ surplus })}
-                      >
-                        Edit
-                      </Button>
-                      <Button
-                        size="small"
-                        color="primary"
-                        onClick={() => this.handleDelete({ surplus })}
-                      >
-                        Delete
-                      </Button>
+                      {/* Only show if it is main page */}
```

Comment:
These comments don't say anything beyond what the code says, so you can remove them here and elsewhere.